dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -145,4 +161,31 @@ function getHealthy(spriteIdArg) {
 function addMonsterBehavior(behaviorFunc) {
   addBehaviorSimple({costume: "all"}, new Behavior(behaviorFunc, []));
 }
+
+function addRecoveryBars() {
+  push();
+  var spriteIds = getSpriteIdsInUse();
+  for (var i = 0; i < spriteIds.length; i++) {
+    var spriteIdArg = {id: spriteIds[i]};
+    var isSick = getProp(spriteIdArg, "sick");
+    if (isSick) {
```

Comment:
nit: we can simplify the nesting here by inverting this conditional and using `continue`

```javascript
if (!getProp(spriteIdArg, "sick")) {
  continue;
}
```